pkg/oc/admin/router/router.go

```diff
@@ -655,6 +665,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out, errout io.Write
 		"STATS_PORT":                            strconv.Itoa(cfg.StatsPort),
 		"STATS_USERNAME":                        cfg.StatsUsername,
 		"STATS_PASSWORD":                        cfg.StatsPassword,
+		"ROUTER_ENABLE_THREADS":                 strconv.FormatBool(cfg.EnableThreads),
+	}
+
+	if cfg.EnableThreads {
```

Comment:
Should an error be returned if a number of threads is specified but threads are not enabled?